sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/display/DisplayData.java

```diff
@@ -796,8 +796,9 @@ private Builder include(Path path, HasDisplayData subComponent) {
         // Don't re-wrap exceptions recursively.
         throw e;
       } catch (Throwable e) {
-        String msg = String.format("Error while populating display data for component: %s",
-            namespace.getName());
+        String msg = String.format(
+            "Error while populating display data for component: %s: %s",
```

Comment:
Should this be "for component '%'s:"? The double colons seem odd.

Reply:
Done.